wagtail_newsletter/rich_text.py

```diff
@@ -0,0 +1,46 @@
+from copy import copy
+from functools import lru_cache
+
+from django.utils.html import escape
+from wagtail.models import Page
+from wagtail.rich_text import EmbedRewriter, LinkRewriter, MultiRuleRewriter, features
+from wagtail.rich_text.pages import PageLinkHandler
+
+
+class LinkHandlerForEmail(PageLinkHandler):
+    @classmethod
+    def expand_db_attributes(cls, attrs):
+        try:
+            page = cls.get_instance(attrs)
+            full_url = page.localized.specific.full_url  # type: ignore
+            return '<a href="%s">' % escape(full_url)
+        except Page.DoesNotExist:
+            return "<a>"
+
+
+@lru_cache(maxsize=None)
+def get_rewriter_for_email():
```

Comment:
it looks to me that this should rather be a private function, as it seems to only be used by `rewrite_db_html_for_email` ?

So maybe we should prefix its name with an underscore, and move it down below the public API of the module?